### DISCUSSION TITLE
Kata terminada

### DISCUSSION THREAD

**Discussion by gersondlarosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#249] [Q&A]
Buena tarde acabo de terminar mi kata donde puedo subirla

**Comment by martinGM05** [comment]
Ya se subio el link en teams.

**Comment by JuanSalvador73** [comment]
Este es el link de la Kata de la pastelería:

https://forms.office.com/pages/responsepage.aspx

**Reply by gersondlarosa** [reply]
Me dice que la liga esta rota voy a ver como la compañera dice
